copier/vcs.py

```diff
@@ -14,7 +14,7 @@
 
 GIT_PREFIX = ("git@", "git://", "git+")
 GIT_POSTFIX = (".git",)
-GIT_VERSION = Version(git("version").split()[-1].replace(".windows", "+windows"))
+GIT_VERSION = Version(re.findall(r".*(\d+\.\d+\.\d+)", git("version"))[0])
```

Comment:
Wouldn't this work? If so, change it please 🙂 

```suggestion
GIT_VERSION = Version(re.findall(r"\d+\.\d+\.\d+", git("version"))[0])
```